client/my-sites/sidebar/sidebar.jsx

```diff
@@ -106,6 +106,10 @@ module.exports = React.createClass( {
 	stats: function() {
 		var site = this.getSelectedSite();
 
+		if ( ! site.capabilities ) {
```

Comment:
Exactly. We still don't know which capabilities the user has for that site for a few milliseconds while the site-list gets fetched again.